aap-common/assembly-aap-command-generator.adoc

```diff
@@ -0,0 +1,43 @@
+ifdef::context[:parent-context: {context}]
+
+[id='aap-command-generator']
+
+
+= Using the Command Generator
+
+The Command Generator is used to generate commands for launching operational playbooks provided by the Ansible-on-clouds operational playbook collection.
```

Comment:
Is Ansible-on-clouds the name of the collection?
If so, maybe enclose it in backticks